packages/sanity/src/core/form/studio/inputs/crossDatasetReference/datastores/getReferenceInfo.ts

```diff
@@ -74,11 +74,7 @@ export function createGetReferenceInfo(context: {
           (candidate) => candidate.type === resolvedDoc._type,
         )
 
-        const previewPaths = [
-          ...(getPreviewPaths(refSchemaType?.preview) || []),
-          ['_updatedAt'],
-          ['_createdAt'],
-        ]
+        const previewPaths = [...(getPreviewPaths(refSchemaType?.preview) || [])]
```

Comment:
Can be simplified to:
```suggestion
        const previewPaths = getPreviewPaths(refSchemaType?.preview) || []
```